ios/FluentUI/Tooltip/TooltipModifiers.swift

```diff
@@ -0,0 +1,135 @@
+    ///   - title: The optional bolded text to be displayed above the message on the new tooltip view.
+    ///   - preferredArrowDirection: The preferrred direction for the tooltip's arrow. Only the arrow's axis is guaranteed; the direction may be changed based on available space between the anchorView and the screen's margins. Defaults to down.
+    ///   - offset: An offset from the tooltip's default position.
+    ///   - dismissMode: The mode of tooltip dismissal. Defaults to tapping anywhere.
+    ///   - isPresented: A binding to a Boolean value that determines whether to present the tooltip. When the user dismisses the tooltip, this value is set to `false`.
+    ///   - onTap: An optional closure used to do work after the user taps
+    @ViewBuilder
+    func fluentTooltip(message: String,
+                       title: String? = nil,
+                       preferredArrowDirection: Tooltip.ArrowDirection = .down,
+                       offset: CGPoint = CGPoint(x: 0, y: 0),
+                       dismissMode: Tooltip.DismissMode = .tapAnywhere,
+                       isPresented: Binding<Bool>,
+                       onTap: (() -> Void)? = nil) -> some View {
+        // Package up all the values to pass through.
+        let values = TooltipAnchorViewValues(
+            message: message,
+            title: title,
+            preferredArrowDirection: preferredArrowDirection,
+            offset: offset,
+            dismissMode: dismissMode,
+            onTap: onTap)
+
+        self.modifier(
+            TooltipModifier(
+                values: values,
+                isPresented: isPresented
+            )
+        )
+    }
+}
+
+// MARK: - Private support for public modifiers
+
+/// Convenience wrapper for the values used to show a `Tooltip`.
+private struct TooltipAnchorViewValues {
+    let message: String
+    let title: String?
+    let preferredArrowDirection: Tooltip.ArrowDirection
+    let offset: CGPoint
+    let dismissMode: Tooltip.DismissMode
+    let onTap: (() -> Void)?
+}
+
+private struct TooltipModifier: ViewModifier {
+    let values: TooltipAnchorViewValues
+    @Binding var isPresented: Bool
+
+    func body(content: Content) -> some View {
+        content
+            .background {
+                TooltipAnchorViewRepresentable(values: values, isPresented: $isPresented)
+            }
+    }
+}
+
+/// `UIView` subclass that serves as an anchor to the `Tooltip`.
+///
+/// Our existing `Tooltip` logic is built entirely around `UIView` anchoring. To reuse this in SwiftUI, we create
+/// a simple `UIView` that acts as this anchor.
+private class TooltipAnchorView: UIView {
+    var values: TooltipAnchorViewValues
+    var isPresented: Binding<Bool>
+
+    init(values: TooltipAnchorViewValues, isPresented: Binding<Bool>) {
+        self.values = values
+        self.isPresented = isPresented
+        super.init(frame: .zero)
+
+        self.backgroundColor = .clear
```

Comment:
nit: shouldn't background color by default be clear?

Reply:
Probably, but I trust nothing 🤠 Think I should remove it?

Reply:
Resolved!